### PR TITLE
Added Yang's solid solution features

### PR DESCRIPTION
**Pull request by WardLT**
Added features from [Yang and Zhang (2012)](http://linkinghub.elsevier.com/retrieve/pii/S0254058411009357), which can predict whether a metallic composition will form a metallic glass or not.

### CODE REVIEW

File: matminer/featurizers/composition.py [hunk 1]
```diff
@@ -1320,3 +1319,123 @@ def citations(self):
...
+
+class YangSolidSolution(BaseFeaturizer):
+    """
+    Mixing thermochemistry and size mismatch terms of Yang and Zhang (2012)
+
+    This featurizer returns two different features developed by
+    .. Yang and Zhang `https://linkinghub.elsevier.com/retrieve/pii/S0254058411009357`.
+    The first, Omega, is related to the balance between the mixing entropy and
+    mixing enthalpy of the liquid phase. The second, delta, is related to the
+    atomic size mismatch betweens the
```
Comment (computron) [inline matminer/featurizers/composition.py]: think some text got cut off here

File: matminer/featurizers/composition.py [hunk 2]
```diff
@@ -1320,3 +1319,123 @@ def citations(self):
...
+        yang delta - Atomic size mismatch term
+
+    References:
+        .. Yang and Zhang (2012) `https://linkinghub.elsevier.com/retrieve/pii/S0254058411009357`.
+    """
+
+    def __init__(self):
+        # Load in the mixing enthalpy data
+        #  Creates a lookup table of the liquid mixing enthalpies
+        mixing_dataset = pd.read_csv(os.path.join(data_dir,
```
Comment (computron) [inline matminer/featurizers/composition.py]: Any reason not to have mixing_dataset be part of "data.py"? 

e.g. you can do:
self.mixing_data = MixingData()

like you did for loading elem_data and Magpie a few lines below

File: matminer/featurizers/composition.py [hunk 3]
```diff
@@ -1320,3 +1319,123 @@ def citations(self):
...
+        radii = self.elem_data.get_elemental_properties(elements,
+                                                        "MiracleRadius")
+        mean_r = PropertyStats.mean(radii, fractions)
+
+        # Compute the mean (1 - r/\bar{r})^2
+        r_dev = np.power(1.0 - np.divide(radii, mean_r), 2)
+        return np.sqrt(PropertyStats.mean(r_dev, fractions))
+
+    def feature_labels(self):
+        return ['Yang Omega', 'Yang delta']
```
Comment (computron) [inline matminer/featurizers/composition.py]: I would stick to a consistent capitalization unless there is good reason not to, e.g.

``Yang omega, Yang delta``

**Comment by computron** [comment]
@WardLT just put some minor code review above

**Comment by WardLT** [comment]
Thanks for the comments.

**Pull request merged**